fabcar/java/src/main/java/org/example/RegisterUser.java

```diff
@@ -36,22 +39,20 @@ public static void main(String[] args) throws Exception {
 		caClient.setCryptoSuite(cryptoSuite);
 
 		// Create a wallet for managing identities
-		Wallet wallet = Wallet.createFileSystemWallet(Paths.get("wallet"));
+		Wallet wallet = Wallets.newFileSystemWallet(Paths.get("wallet"));
 
 		// Check to see if we've already enrolled the user.
-		boolean userExists = wallet.exists("appUser");
-		if (userExists) {
+		Identity user = wallet.get("appUser");
```

Comment:
Since we don't use the value returned here anywhere else, I wonder if it would be clearer to combine this line and the one below rather than the (potentially confusing) use of the `user` variable here and again further down in the code for different values? For example:
```
if (wallet.get("appUser") != null) {
```

Reply:
Ok, and I've changed EnrollAdmin the same way.